src/pages/events/index.js

```diff
@@ -0,0 +1 @@
+export { default, getServerSideProps } from '@/pages/redirect';
```

Comment:
funky 😄

Reply:
Why? We're doing this on all our pages :)